pkg/maestro/client.go

```diff
@@ -50,73 +48,32 @@ type Client struct {
 	client   *http.Client
 }
 
-func (m Client) CreateTask(owner, queue, payload string) (string, error) {
+func (m Client) CreateTask(owner, queue, payload string, options ...createTaskOptions) (string, error) {
+	opt := MergeCreateTaskOptions(options...)
 	httpPayload := struct {
-		Owner   string `json:"owner"`
-		Queue   string `json:"queue"`
-		Retries int    `json:"retries"`
-		Timeout int    `json:"timeout"`
-		Payload string `json:"payload"`
+		Owner        string `json:"owner"`
+		Queue        string `json:"queue"`
+		Retries      int    `json:"retries"`
+		Timeout      int    `json:"timeout"`
+		Payload      string `json:"payload"`
+		StartTimeout *int   `json:"startTimeout,omitempty"`
+		ExecutesIn   *int   `json:"not_before,omitempty"`
```

Comment:
Un peu verbose mais vu que t'as renommé et que c'est des int...

```suggestion
		StartTimeoutSecs *int   `json:"startTimeout,omitempty"`
		ExecutesAfterSecs   *int   `json:"not_before,omitempty"`
```

Reply:
Pas fan non plus des noms mais j'ai en fait utilisé ceux de la lib Python et de maestro
Mieux vaut uniformiser même si je suis d'accord que ces noms ne sont pas les plus explicites